advanced/Templates/pihole-FTL.service

```diff
@@ -60,16 +89,16 @@ stop() {
 
     if is_running; then
       echo "Not stopped; may still be shutting down or shutdown may have failed, killing now"
-      pkill -xo -9 "pihole-FTL"
-      exit 1
+      pkill -9 -F "${FTL_PID_FILE}"
+      exit 0
```

Comment:
No need to explicitly exit here explicitly, if it shall be a non-failure exit. Also, if a positive exit code is returned, hence the service is shown as stopped, the cleanup steps below shouldn't be skipped.
```suggestion
```
But actually doesn't it make sense to keep the service stop failing if killing it fails, so users get feedback about that?